### ISSUE TITLE
Close button is ineffective on Windows

### ISSUE THREAD

**Issue by PhiLhoSoft** [sindresorhus/anatine#17]
I just downloaded the Windows version of Anatine, using it on Windows 7 Professional.

First remark: it might be useful to have the classical About (the application) dialog showing the current version. Yes, I can go where I installed (unzipped) the application, open the `version` file and see _v0.37.8_, but a classical / familiar / quick way of showing the version, and some credit, would be nice.

I am trying Anatine because I am tired that going back to where I left in the timeline eats megabytes and brings my browsers to their knees. At least, I can restart Anatine if it becomes too memory hungry!

But then, I noticed that clicking on the rightmost, big red button with a white cross on the window frame (the so called Close button) doesn't close Anatine at all!
I still see the two processes in memory, waiting for a restart.
(Three processes. I see now a GPU process.)
Launching Anatine again spawns yet another process but it sees the current ones so it auto-closes cleaning, leaving the focus to the previous one.

When I go to File > Quit, the processes effectively close.
BTW, I appreciate that restarting again is very fast. Much faster than launching Atom! ;-)

So, TL;DR: please, ensure you close the Anatine processes when the user clicks on the Close button, not just its window.

**Comment by sindresorhus** [comment]
It doesn't quit because it's minimized to the tray. I've clarified this in the readme now: https://github.com/sindresorhus/anatine/commit/0f6ded55ae1089c345f37bbed9a039c7919a9f59 This is so that you can be notified about new DM when https://github.com/sindresorhus/anatine/issues/2 is fixed.

**Comment by PhiLhoSoft** [comment]
Ah, it is always confusing... Some applications minimize to the tray on the Minimize button (eg. FooBar2000), others do that on the Close button (eg. Skype).
That's OK then, feel free to close the issue.

PS: sub-suggestion - if possible, would be interesting to show the number of new tweets in the tooltip of the tray icon.
PPS.: maybe a "Minimize to tray" menu item above Quit might be a hint of this capability...
PPPS.: Why the window title is "Twitter" while the tray icon tooltip shows "Anatine"? :wink:

**Comment by sindresorhus** [comment]
About menu added.

**Comment by sindresorhus** [comment]
> if possible, would be interesting to show the number of new tweets in the tooltip of the tray icon.

Open a new issue and we can discuss there.

> maybe a "Minimize to tray" menu item above Quit might be a hint of this capability...

Nah, that's non-standard. I'd rather document the behavior.

> Why the window title is "Twitter" while the tray icon tooltip shows "Anatine"? 

Fixed in https://github.com/sindresorhus/anatine/commit/7f3b57e2a77b360a34d8000dd19bcbfa8d433ba9. I didn't realize this happened as the title is not visible on OS X, which I use.

-

Thanks for all the feedback. Keep it coming :)
